draft-ietf-idr-link-bandwidth.xml

```diff
@@ -187,6 +220,12 @@
           any way.</t>
         </section>
       </section>
+      <section title="Link bandwidth community aritmetics and BGP multipath">
+      <t>In BGP multi-path ECMP enviroment, send or re-advertised link-bandwidth
```

Comment:
Should we recommend WECMP to be performed only if all contributing paths contain LBW community?

Reply:
> Should we recommend WECMP to be performed only if all contributing paths contain LBW community?

I think Rafal added this section to just say that things like 'bandwidth aggregation' are out of scope of this draft. handled in some other draft bess. 

You may instead add the above suggested text as general LBW procedure elsewhere in this draft.

Reply:
Sure. I will take this up and update the draft after we merge these changes.